docs/API.md

```diff
@@ -84,6 +84,9 @@ curl -v \
 {
   "uid": "6d940dd41e636cc156074109b8092f96",
   "email": "user@example.domain",
+  "locale": "en-US",
+  "amr_values": ["pwd", "otp"],
```

Comment:
I'm torn here, between using "amr_values" for consistency with OAuth specs, versus "amrValues" for consistency with camel-casing of "avatarDefault" and "displayName".  I think I'll probably wind up favouring local consistency and switching these to camel-case.